moj_auth/tests/test_api_client.py

```diff
@@ -24,13 +24,11 @@ def test_invalid_credentials(self):
             content_type='application/json'
         )
 
-        # authenticate, should return None
-        token = authenticate(
-            'my-username', 'invalid-password'
+        # authenticate, should raise Unauthorized
```

Comment:
You could remove this comment and rename the method `test_invalid_credentials_is_unauthorized` or `_raises_unauthorized`, maybe?